pkg/docker/docker.go

```diff
@@ -59,7 +59,7 @@ type Client interface {
 	StartBuildkitd(ctx context.Context, tag, name, mirror string) (string, error)
 
 	IsRunning(ctx context.Context, name string) (bool, error)
-	IsCreated(ctx context.Context, name string) (bool, error)
```

Comment:
It's better to add a new method `Exists` and mark `IsCreated` deprecated.

Reply:
We are still in alpha, thus I think it also works although we remove it directly.

There is no project that relies on us, currently.